test/ApplicationSamplerTest.cpp

```diff
@@ -525,3 +547,25 @@ TEST_F(ApplicationSamplerTest, cpu_progress)
         .WillOnce(Return(expected));
     EXPECT_EQ(expected, m_app_sampler->cpu_progress(1));
 }
+
+TEST_F(ApplicationSamplerTest, sampler_cpu)
+{
```

Comment:
Comments please to explain the numbering here between physical cores and linux logical cpus.